nomad/local/start_local_registry.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Starts up a local docker registry.
+# This is required for local nomad deployments
+
+docker run -d -p 5000:5000 --name registry registry:2.7
```

Comment:
Would there be a way to tie this into our makefile? 

So that it'd look something like
```
.PHONY: run-registry
run-registry:
 	# start local registry *if it hasn't already been started* 

.PHONY: push-local
push-local: run-registry ## Push Grapl container to local registry
	REGISTRY=localhost:5000 docker-compose --file=docker-compose.build.yml push
```

it feels odd that there's now a manual step before we can `make` anything

Reply:
Oh yea that's a great idea. I don't know why that didn't cross my mind. I'll migrate that over right away.